### DISCUSSION TITLE
2024/10/28/Goals-for-24-25-School-Year/

### DISCUSSION THREAD

**Discussion by giscus** [C0DET1GER/c0det1ger.github.io#2] [General]
# 2024/10/28/Goals-for-24-25-School-Year/

Seems a bit too late, but whatever

https://c0det1ger.github.io/2024/10/28/Goals-for-24-25-School-Year/

**Comment by PsychoPinkQ** [comment]
Diao.
LGM when?

**Reply by the-coding-pooh** [reply]
Why is codet1ger so orz

**Reply by C0DET1GER** [reply]
Why is pooh so orz

**Comment by iamnoob2004** [comment]
you are the goat

**Reply by C0DET1GER** [reply]
teach me how to be LGM...

**Comment by Algorithmic149** [comment]
those who admit U4
